controllers/dogu_manager.go

```diff
@@ -41,22 +53,54 @@ func NewDoguManager(client client.Client, operatorConfig *config.OperatorConfig,
 		return nil, fmt.Errorf("failed to validate key provider: %w", err)
 	}
 
-	installManager, err := NewDoguInstallManager(client, operatorConfig, cesRegistry, eventRecorder)
+	// get AND validate config in one yay
+	util.NewAdditionalImageGetter(client, operatorConfig.Namespace).ImageForKey(nil, "")
```

Comment:
🚫 **[checkstyle]** <errcheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Error return value of `(*github.com/cloudogu/k8s-dogu-operator/controllers/util.additionalImageGetter).ImageForKey` is not checked

Reply:
done with 9e14098e